code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -2,11 +2,13 @@
 	name = "wheelchair"
 	desc = "Now we're getting somewhere."
 	icon_state = "wheelchair"
-	anchored = 0
-	buckle_movable = 1
-	movement_handlers = list(/datum/movement_handler/deny_multiz, /datum/movement_handler/delay = list(2), /datum/movement_handler/move_relay_self)
+	anchored = FALSE
+	buckle_movable = TRUE
+	movement_handlers = list(/datum/movement_handler/deny_multiz, /datum/movement_handler/delay = list(5), /datum/movement_handler/move_relay_self)
 	movable_flags = MOVABLE_FLAG_NONDENSE_COLLISION
-	var/driving = 0
+
+	var/item_form_type = /obj/item/wheelchair_kit
```

Comment:
Not a blocker, but I'm starting to think we should really make 'turns into an item for portability' an extension.